hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -502,6 +502,121 @@
         ]
       }
     }
+    { name: "phy_pins_sense",
+      desc: "USB PHY Pin Sense",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      fields: [
+        {
+          bits: "0",
+          name: "rx_dp_i",
+          desc: "USB D+ input."
+        }
+        {
+          bits: "1",
+          name: "rx_dn_i",
+          desc: "USB D- input."
+        }
+        {
+          bits: "2",
+          name: "rx_d_i",
+          desc: "USB Differential input."
+        }
+        {
+          bits: "8",
+          name: "tx_dp_o",
+          desc: "USB D+ output (readback)."
+        }
+        {
+          bits: "9",
+          name: "tx_dn_o",
+          desc: "USB D- output (readback)."
+        }
+        {
+          bits: "10",
+          name: "tx_d_o",
+          desc: "USB Differential output (readback)."
+        }
+        {
+          bits: "11",
+          name: "tx_se0_o",
+          desc: "USB SE0 output (readback)."
+        }
+        {
+          bits: "12",
+          name: "tx_oe_o",
+          desc: "USB OE output (readback)."
+        }
+        {
+          bits: "13",
+          name: "suspend_o",
+          desc: "USB suspend output (readback)."
+        }
+        {
+          bits: "16",
+          name: "pwr_sense",
+          desc: "USB power sense signal."
+        }
+      ]
+    }
+    { name: "phy_pins_drive",
+      desc: "USB PHY Pin Drive",
```

Comment:
Similarly add "(for testing)"

Reply:
same here: "I have extended the description to mention the purpose of the registers (debugging and chip testing)."